packages/logs/src/domain/logger.ts

```diff
@@ -39,29 +52,53 @@ export class Logger {
   }
 
   @monitored
-  log(message: string, messageContext?: object, status: StatusType = StatusType.info) {
-    this.handleLogStrategy({ message, context: deepClone(messageContext) as Context, status }, this)
+  log(message: string, messageContext?: object, status: StatusType = StatusType.info, error?: Error) {
+    let context: Context
+
+    if (status === StatusType.error || (error !== undefined && error !== null)) {
+      // Always add origin if status is error (backward compatibility - Remove in next major)
+      const errorContext = ({ origin: ErrorSource.LOGGER } as LogsEvent['error'])!
+
+      if (error !== undefined && error !== null) {
+        const stackTrace = error instanceof Error ? computeStackTrace(error) : undefined
+        const rawError = computeRawError({
+          stackTrace,
+          originalError: error,
+          nonErrorPrefix: PROVIDED_ERROR_MESSAGE_PREFIX,
+          source: ErrorSource.LOGGER,
+          handling: ErrorHandling.HANDLED,
+          startClocks: clocksNow(),
+        })
+
+        errorContext.stack = rawError.stack
+        errorContext.message = rawError.message
+        if (rawError.type) {
+          errorContext.kind = rawError.type
+        }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could remove this `if` and let the serialization remove the undefined property

Reply:
👌